svelte/components/Nft/NftsListLines.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import type { Readable } from "svelte/store";
+  import type { NftType } from "lib/ktypes";
+
+  import { getNetwork } from "lib/kconfig";
+
+  import NftLine from "./NftLine.svelte";
+
+  /////////////////////////////////////////////////
+  //  <NftListLines {chainId} {account} {nfts} {platform}? />
+  // Display NFTs List
+  /////////////////////////////////////////////////
+  export let chainId: number;
+  export let account: string;
```

Comment:
should have same props than Grid, i.e. tokenID also ?

Reply:
actually, in the line display we don't need the tokenID because the detail of an nft is shown by the toggle of the line.
in the grid mode i need it to open one Nft, but as we said it's perhaps not the better way...